conference/cfp/views.py

```diff
@@ -8,13 +8,15 @@
 from django.views.generic import CreateView
 from djmail.template_mail import MagicMailBuilder
 
-from .forms import SubmissionForm
-from .models import Submission
-
 
 class SubmissionView(CreateView):
-    form_class = SubmissionForm
-    model = Submission
+
+
+    def __init__(self, form_class, model, *args, **kwargs):
```

Comment:
We don't need this, ``as_view()`` automatically set the attributes on the class instance